ABOUT.md

```diff
@@ -0,0 +1,51 @@
+
+# What is MineColonies?
+Here, we would like to give you an overview of what our mod is.
```

Comment:
This is an overview of what our mod is about.

Reply:
Sounds perfect!